keadm/cmd/keadm/app/cmd/edge/batchjoin.go

```diff
@@ -0,0 +1,381 @@
+// Copyright 2024 WeiXingEsther
```

Comment:
https://github.com/kubeedge/kubeedge/blob/master/edge/cmd/edgecore/app/options/options.go#L15

Add copyright like this :) Thanks.